README.md

```diff
@@ -165,7 +165,7 @@ streams:
 | Entity                                    | Description                                                              | Enabled by default |
 | ----------------------------------------- | ------------------------------------------------------------------------ | ------------------ |
 | `switch.*_blinddetect_enabled`            |                                                                          | No                 |
-| `select.*_day_night_color`                | You can control camera's lights.                                         | No                 |
+| `select.*_whith_light`                    | You can control camera's lights.                                         | No                 |
```

Comment:
Hey @dbuezas 

where is it defined?
my current code in select.py is still using the old name `select.camera_varanda_day_night_color`.

Reply:
Ok after I restart, it is the new code 😄

Reply:
Try removing and reading the cameras

Reply:
Oh, yes, you need to restart after any code change, no exceptions 👍

Reply:
Now it is showing on entities.
<img width="742" alt="Screenshot 2023-12-19 at 11 39 52" src="https://github.com/dbuezas/icsee-ptz/assets/6358755/2494bb8d-79f0-42ac-ad83-c4023dd50e51">

But not here:
<img width="358" alt="Screenshot 2023-12-19 at 11 40 10" src="https://github.com/dbuezas/icsee-ptz/assets/6358755/9427d0ac-28af-4898-aa5b-575bf8cbb252">

Any idea why? am I missing something?

Reply:
I found the reason 😄